src/price/PriceHelper.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+declare(strict_types=1);
+
+namespace hiqdev\php\billing\price;
+
+use Money\Currencies\ISOCurrencies;
+use Money\Currency;
+use Money\Money;
+use Money\Parser\DecimalMoneyParser;
+
+class PriceHelper
+{
+    public static function buildMoney(string $price, string $currency): ?Money
+    {
+        if (!is_numeric($price)) {
+            throw new \InvalidArgumentException('price of the Money must be numeric');
+        }
+        if (!self::checkFloat($price)) {
+            return new Money($price, new Currency($currency));
+        }
+        if (!is_int($price) && ($price * 100) == ((int) ($price * 100))) {
+            return (new DecimalMoneyParser(new ISOCurrencies()))->parse($price, new Currency($currency));
+        }
+        return new Money(
+            (int) ($price * self::calculatePriceRate($price)),
+            new Currency($currency)
+        );
+    }
+
+    public static function calculatePriceRate(string $price): int
```

Comment:
`Price Rate` is not a self-descriptive name. It's also misleading.

Rate – is about a ratio. A ratio can be non-integer. I'm thinking about better naming.

Maybe Multiplier? @hiqsol